core/client/fs/src/main/java/alluxio/client/block/stream/BlockWorkerClient.java

```diff
@@ -57,6 +58,8 @@ public static BlockWorkerClient create(UserState userState, GrpcServerAddress ad
         throws IOException {
       try {
         return new DefaultBlockWorkerClient(userState, address, alluxioConf);
+      } catch (UnavailableException e) {
```

Comment:
Could you help me to figure out where and when the alluxio.client.file.FileSystemContext.acquireBlockWorkerClientInternal  throws the UnavailableException?

Reply:
In acquireBlockWorkerClientInternal function, from acquire it will create a new resource. Then DefaultBlockWorkerClient will build a channel. Finally startAuthenticatedChannel function will throw an UnavailableException